codecs/opus/src/main/java/org/restcomm/media/codec/opus/Encoder.java

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
Update disclaimer. Similar to https://github.com/RestComm/mediaserver/blob/master/asr/asr-api/src/main/java/org/restcomm/media/asr/AsrEngine.java#L1

Reply:
Fixed